src/RawModelExportList.tsx

```diff
@@ -4,73 +4,90 @@ import Divider from '@mui/material/Divider'
 import Stack from '@mui/material/Stack'
 import Typography from '@mui/material/Typography'
 import Box from '@mui/system/Box'
-import TextField from '@mui/material/TextField'
-import IconButton from '@mui/material/IconButton'
-import CopyIcon from '@mui/icons-material/ContentCopyTwoTone'
-import Tooltip from '@mui/material/Tooltip'
 
 import { useGetModelById, useGetModelVersions } from '../data/model'
 import { Deployment } from '../types/interfaces'
-import CopiedSnackbar from './common/CopiedSnackbar'
 import EmptyBlob from './common/EmptyBlob'
+import Button from '@mui/material/Button'
+import axios from 'axios'
+import Snackbar from '@mui/material/Snackbar'
 
-const DeploymentVersion = ({ text, type }: { text: string; type: string }) => {
+const RawModelExportList = ({ deployment }: { deployment: Deployment }) => {
+  const { model } = useGetModelById(deployment.model.toString())
+  const { versions } = useGetModelVersions(model?.uuid)
   const [openSnackbar, setOpenSnackbar] = React.useState(false)
 
-  const handleButtonClick = () => {
-    navigator.clipboard.writeText(text)
-    setOpenSnackbar(true)
+  const handleClose = (_event, reason) => {
+    if (reason === 'clickaway') {
+      return
+    }
+    setOpenSnackbar(false)
   }
 
-  return (
-    <Stack spacing={2} direction='row' sx={{ p: 1 }}>
-      <TextField
-        sx={{ width: 500 }}
-        label={type + ' path'}
-        defaultValue={text}
-        inputProps={{ readOnly: true }}
-        variant='filled'
-      >
-        {text}
-      </TextField>
-      <Tooltip title='Copy' arrow placement='bottom'>
-        <IconButton aria-label='delete' onClick={handleButtonClick}>
-          <CopyIcon />
-        </IconButton>
-      </Tooltip>
-      <CopiedSnackbar {...{ openSnackbar, setOpenSnackbar }} />
-    </Stack>
-  )
-}
+  const downloadFile = async (version: string, fileType: string) => {
```

Comment:
We don't want to download files like this.  We want to use an `a href` so that the user can either click it to have it download, or right click -> copy URL to get the URL.

Reply:
Done